activity.cc

```diff
@@ -129,4 +132,99 @@ ActivityLog PolluteDataCache::GetActivityLog() {
   return alog;
 }
 
+// Activity: PolluteInstructionCache
+
+// The boost library (https://github.com/nelhage/rules_boost) is used to
+// create a large 2D array of functions with names -
+//
+// DummyFunc_0_0, DummyFunc_0_1, ..., DummyFunc_0_{POLLUTE_ICACHE_LOOP_SIZE-1},
+// ...
+// ...
+// ...
+// DummyFunc_{POLLUTE_ICACHE_LOOP_SIZE-1}_0, ..., DummyFunc_{POLLUTE_ICACHE_LOOP_SIZE-1}_{POLLUTE_ICACHE_LOOP_SIZE-1}.
+// Each of these functions returns an int.
+//
+// This 2D array was created because BOOST_PP_REPEAT() macro can
```

Comment:
"foo was created" is passive voice. "we create foo" is better

Reply:
Done